gen_map_test can flake

```
/Users/rick/code/src/github.com/solo-io/gloo/pkg/cliutil/nsselect/gen_map_test.go:20

  Expected
      <[]string | len:3, cap:4>: ["ns2, s3", "ns1, s1", "ns1, s2"]
  to equal
      <[]string | len:3, cap:3>: ["ns1, s1", "ns1, s2", "ns2, s3"]

  /Users/rick/code/src/github.com/solo-io/gloo/pkg/cliutil/nsselect/gen_map_test.go:22
------------------------------


Summarizing 1 Failure:

[Fail] Generate Options [It] should create the correct Secret options and map
/Users/rick/code/src/github.com/solo-io/gloo/pkg/cliutil/nsselect/gen_map_test.go:22

Ran 1 of 1 Specs in 0.001 seconds
FAIL! -- 0 Passed | 1 Failed | 0 Pending | 0 Skipped
--- FAIL: TestGit (0.00s)
FAIL
```

--- a/pkg/cliutil/nsselect/gen_map_test.go
+++ b/pkg/cliutil/nsselect/gen_map_test.go
@@ -17,10 +17,9 @@ var _ = Describe("Generate Options", func() {
 		Secrets:   []string{"s3"},
 	}
 
-	// https://github.com/solo-io/gloo/issues/210
-	PIt("should create the correct Secret options and map", func() {
+	It("should create the correct Secret options and map", func() {
 		genOpts, resMap := generateCommonResourceSelectOptions("secret", nsrMap)
-		Expect(genOpts).To(Equal([]string{
+		Expect(genOpts).To(ConsistOf([]string{
 			"ns1, s1",
 			"ns1, s2",
 			"ns2, s3",
@@ -50,6 +49,6 @@ var _ = Describe("Generate Options", func() {
 				Namespace: "ns2",
 			},
 		}
-		Expect(resMap).To(Equal(expectedMap))
+		Expect(resMap).To(BeEquivalentTo(expectedMap))
 	})
 })

